### PR TITLE
Completed html-forms exercise - Andrew Almeida

### PR DESCRIPTION
**Pull request by Andrew-M-A**
ezgif com-gif-maker (10)

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title> HTML Forms </title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+
+    <p>Email</p>
```
Comment (codingcodymiller) [inline html-forms/index.html:10]: This should be the `label` element, it is the text labelling the form control element.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <label for="email">
+      <input type="text" id="email" name="email">
+    </label>
+
+    <p>Why is coding cool?</p>
```
Comment (codingcodymiller) [inline html-forms/index.html:15]: This should be the `label` element, it is the text labelling the form control element.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <input id="CSS" type="radio" name="language" value="CSS">
+    <label for="CSS"> CSS </label>
+    <input id="JavaScript" type="radio" name="language" value="JavaScript">
+    <label for="JavaScript"> JavaScript </label>
+
+    <p>Would you like to sign up for more information?</p>
+    <input type="checkbox" name="signUp">
+    <label for="signUp"> Sign me up! </label>
+
+    <p>Comments</p>
```
Comment (codingcodymiller) [inline html-forms/index.html:37]: This should be the `label` element, it is the text labelling the form control element.

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <label for="comments"></label>
+    <textarea name="comments" id="comments" cols="30" rows="10"></textarea>
+
+    <div>
+      <label for="submit"></label>
```
Comment (codingcodymiller) [inline html-forms/index.html:42]: There is no text labelling this button, so no `label` is necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Andrew-M-A** [comment]
html forms

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**